style.css/thirdsectionStyle.css

```diff
@@ -0,0 +1,113 @@
+.third-section {
+  text-align: center;
+}
+
+.title-third-section {
+  background-color: red;
```

Comment:
There is no red in the design please find the right hex in the Figma and replace all red with the correct value. If you need help with it please reach out :)